src/bridge/jsHelpers.js

```diff
@@ -126,7 +127,12 @@ export const makeScanAccounts = (
 
       if (balance.isNaN()) throw new Error("invalid balance NaN");
 
-      if (operations.length === 0 && balance.isZero()) {
+      const isAccountEmpty =
+        currency.id === "tron"
+          ? get(accountShape, "resources.bandwidth.freeLimit", 0) === 0
```

Comment:
BTW i'm not super fan of using get() much for parts we're "safe" on, I mean we should always have .resources.bandwidth.freeLimit right?

flow type won't cover these